src/Modules/Tracking/Integrations/Bilmur.php

```
@@ -35,9 +43,64 @@ public function is_active(): bool {
+	 * This filter is provided by wpcomsh and allows us to inject custom properties
+	 * into the Bilmur data without duplicating the script or meta tag.
+	 *
+	 * @since   1.2.0
+	 * @version 1.2.0
+	 *
+	 * @param array<string, string> $kv      The existing key-value pairs.
+	 * @param string                $service The bilmur service name.
+	 *
+	 * @return array<string, string> The modified key-value pairs.
+	 */
+	public static function filter_wpcomsh_rum_kv( array $kv, string $service ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$service'. (undefined)

(UnusedFormalParameter)